src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -29,12 +29,13 @@
 		@click="handleClick"
 		@keydown.enter="handleClick">
 		<img v-if="(!isLoading && !failed)"
-			:class="previewSizeClass"
+			:class="previewImageClass"
 			class="file-preview__image"
 			alt=""
 			:src="previewUrl">
 		<img v-if="!isLoading && failed"
-			:class="previewSizeClass"
+			class="mimeicon"
```

Comment:
Is there any case not covered by adding it in `previewImageClass` or can be the explicit class removed?

Reply:
will add to the method